README.md

```diff
@@ -25,24 +25,41 @@ via GitHub pull requests or issues.
 Install the XML extension by RedHat. The exention ID is
 [`redhat.vscode-xml`](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-xml).
 
-### Support for GeekoDoc's RELAX NG schema
-
-To let VSCode be aware of GeekoDoc, you need to associate XML files with the
-GeekoDoc schema. Add the following to `settings.json` and save the file in your
-user directory or the project's workspace:
-
-```json
-"xml.fileAssociations": [
-    {
-        "pattern": "**/*.asm.xml",
-        "systemId": "file:///usr/share/xml/docbook/schema/rng/5.2/assemblyxi.rng"
-    },
-    {
-        "pattern": "**/*.xml",
-        "systemId": "file:///usr/share/xml/geekodoc/rng/geekodoc-latest-flat.rng"
-    }
-],
-```
+### Support for RELAX NG schema
+
+To let VSCode be aware of GeekoDoc and DocBook assembly schema, you have two options:
+
+- Insert an XML model definition at the top of the XML file *before* the root
```

Comment:
Some comments/questions:

* Should we list the XML file extensions **before** the XML model definition? I think, the first is easier to use and should be the preferred method, thus having it listed as the first item.
* Should we recommend the XML file extensions over the XML model? I think so. A "this is the preferred method" or "recommended" after the first sentence would probably be sufficient.
* Mention that the XML model definition is only needed, if you want/need to deviate from the preferred config with the XML file extension.

Reply:
i had the feeling that xml-model is a cleaner solution, isn't it?
I can revert the order if you think it makes more sense of course :-)

Reply:
Done.